_distro_map.yml

```diff
@@ -9,3 +9,16 @@ openshift-enterprise:
     enterprise-4.1:
       name: '4.1'
       dir: container-platform/4.1
+openshift-aro:
+  name: Azure Red Hat OpenShift
+  author: OpenShift Documentation Project <openshift-docs@redhat.com>
+  site: commercial
+  site_name: Documentation
+  site_url: https://docs.openshift.com/
+  branches:
+    enterprise-3.11:
+      name: '3'
+      dir: aro/3
+    enterprise-4.3:
+      name: '4'
+      dir: aro/4
```

Comment:
* does this break all existing links by moving the v3 docs from /aro to /aro/v3? - I presume that it does and that it's inevitable?
* it would be probably be wise to have a v3/v4 version select drop-down on https://docs.openshift.com/ like OCP has if possible

Reply:
This update will break existing links. We plan to redirect existing links to aro/3 urls.

I think we can do a button or dropdown. My first intuition was button to match what we do with dedicated. I think if we eventually have 3 options a drop-down is friendlier.